composer.json

```diff
@@ -40,6 +40,7 @@
     "require": {
         "php" : "^7.3",
         "ext-curl": "*",
+        "ext-intl": "*",
```

Comment:
Hmm. I wonder how we can handle cases where this is not available? The required PHP extensions here will only be noticeable by devs who are running composer, rather than the average stable install. 🤔

Reply:
We could add it to the version.php list of extensions, but it'd be nice to be able to not require the extension if possible, and handle a fallback case internally in the Format class if possible.

Reply:
The [strftime](https://www.php.net/manual/en/function.strftime.php) documentation cited [date()](https://www.php.net/manual/en/function.date.php) and [intldateformatter::format()](https://www.php.net/manual/en/intldateformatter.format.php) as the alternatives. The class [IntlDateFormatter](https://www.php.net/manual/en/class.intldateformatter.php) comes with the the [intl](https://www.php.net/manual/en/book.intl.php) extension. So it seems to be the safest path to replace strftime for us.

Reply:
Yes, I think it's a good replacement, I wonder about how to gracefully handle the additional library without needing to requite ext-intl for all systems (especially those with Gibbon already installed that are updating, particularly XAMPP systems seem to be very minimal on PHP extensions by default).

Reply:
[This article](https://arjunphp.com/install-intl-extension-xampp/) states that XAMPP has already shipped with Intl extension. You just need to edit php.ini and uncomment a certain line to have it enabled. It's a bit of trouble, but its not very hard to do so.

I've also checked Packagist. There is one [strftime replacement library](https://packagist.org/packages/php81_bc/strftime) but it's also written with IntlDateFormatter.

P.S. We can use the [php81_bc/strftime](https://packagist.org/packages/php81_bc/strftime) library for all the functions exposing the strftime formatting string. That way we can be lazy with the parameter rewrite.